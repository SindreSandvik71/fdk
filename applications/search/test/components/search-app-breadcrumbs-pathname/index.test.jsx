import React from 'react';
import { shallow } from 'enzyme';
import PurePathNameBreadcrumb from '../../../src/components/search-app-breadcrumbs-pathname';

test('should render PurePathNameBreadcrumb  correctly', () => {
  const defaultProps = {
    pathname: 'about'
  };
  const wrapper = shallow(<PurePathNameBreadcrumb {...defaultProps} />);
  expect(wrapper).toMatchSnapshot();
});

test('should render PurePathNameBreadcrumb correctly with no pathname', () => {
  const defaultProps = {};
  const wrapper = shallow(<PurePathNameBreadcrumb {...defaultProps} />);
  expect(wrapper).toMatchSnapshot();
});
