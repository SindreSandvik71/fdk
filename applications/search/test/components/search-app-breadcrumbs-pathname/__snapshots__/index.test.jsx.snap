// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PurePathNameBreadcrumb  correctly 1`] = `<span />`;

exports[`should render PurePathNameBreadcrumb correctly with no pathname 1`] = `<span />`;
